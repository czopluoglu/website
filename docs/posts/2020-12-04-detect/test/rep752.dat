11011101010000100010
10101111111001110111
11010110111110101100
10111011011011110111
01010001011110101010
00000000000110100101
00111001011011001010
10010100000100110000
10101101111001010010
11010101100100110110
10000101001010110111
00101111010000011111
01000000010000110110
00000001110000100011
00011101001111100110
10111101010110010111
00110101011001000110
01000001101101000111
10100111111100011101
10110110001011000011
11011010001000101000
00101101011100100100
10100101001010101011
11001000101000001110
11110111101010101100
11101111101111101011
10011101000100100010
00100001011101110101
11110111111111111001
00000001110010001001
10101101001110110000
10011001011010000001
01000000100110010111
11001000100011001101
11000011011000001001
10001011110111100001
11011111011001100111
11111011111011110111
10110001001011011111
10101001011100010011
00001101101100110110
01101000001000111111
00010000000100100111
11010111110001111110
11011100100110100110
10011001111000010100
00101100101010111110
01011010111011001110
01110111111010000011
11000011111010000111
01011100010110000010
10010111111111111110
01001001111011111111
10001101110010101011
01010001111110110110
10100101101110111111
00100111111111111110
11101111111101000110
11111111111011000011
00000001011111111111
11001010001111110111
10011001111000100111
11100101011110011110
11110110111010100001
10100000011111000111
10101001001101100111
01110011001101110111
11110101111110010100
10111000110101100010
10000000000001000110
11110110011010111111
00110011111101000010
10001000101111101111
00001011110101011011
00000001011010100110
01111010101010001011
10011101011000110111
01000001011110101110
10010010011101101111
00010101000101100110
00000111010100100100
10010011101001111011
00010100010010101001
10011000000010000000
00111001001011101010
10101111011111111011
01101001011000110100
01110101111111110011
10001000000000100001
10000100011010010111
00100101001111101111
11101001001010100011
10111111111010111111
11100101100001110110
11111111111111110111
10101100101100001011
11101101111110001111
00101001101010110110
10001101011111100110
01101101001010000110
11110101001011110111
00000000101111101010
11000000000110111001
01111101101000000000
00110001000000110010
01100000000011010011
00001000001010110010
10011101111111110111
11111110111001110110
11110110110000100010
00100001011001111000
01001111110100101101
00000100001010000010
01010101011010000001
10110000001010101101
00000000000100110000
11101000011010100111
11100100101001101110
11000111111111111110
11111101111010100010
11001001001010111111
00101010110111111110
10000000101000001010
00100000000000000011
11010001111101001101
00100000000100000000
10000100011111110010
11001000100110100010
00001011000010100101
00001111101111001110
01001000010110000110
11100101001110101110
01000001111010110111
11010101101110011111
11100001101101111111
10100100011111111111
00000000001110100000
01001001101100101010
00101101010011010110
01000110111010100101
01100001000000000010
11001010101000000111
11011111110001001000
00110000011110110011
00101010000010010011
11111001011010100110
11100111111101011111
11011100110100111111
10001001100101101111
11010111110010010110
10111111001010100101
01011001101001000110
00010010011110111001
00000001001011110010
01010111111011010110
11001000101101101101
11010001110100001110
10110111001101111110
01001010011001101111
10111000011010101111
00110111001101111011
10100001001000010110
11010110010010001110
00000101010110100110
10011101111000111101
01011100101101001000
11111111111111110110
00010000011111100001
11100111100110111011
10000000011100100100
01111111111111111111
00011101010101000100
00011001011000011111
11101110101011100000
00000001001100100111
00010011101000000111
11010011111010101111
01110001111110111111
10000001011110001011
01011011011100111111
11111011111111111011
00001111110111011111
00000100100010011110
11000111011100101110
10000001001110111110
00011000111110100101
11010111010000000010
00101001001111111110
10001101000101111111
11100001100011101110
10001110011010100001
10101001111111001110
11011111010111010010
00110011100100100110
10111101111110011111
00001001111010101011
10010001111110011111
11110101110110000010
00000111111111111111
11110111101010111110
01100001101000001100
11010110001100111111
11101111111010101111
10010100001110101110
10100101001100100011
10011000011111000111
11101111001000101110
00110000111111101111
00000100011110001111
00100001001010111110
10011110110000000110
11101100111010111111
01000000011000100010
00001000001000000110
00101101011101101110
00000000111010001011
00001000001000101100
10010101101010100111
00101100011010100110
00100101001001101011
00011000011111011110
00010001000000011010
01100010011110101101
11101101011000001011
10110110011100110111
11011101110011001111
01000001011110100100
10001101001000100011
11111110111111111100
01010000111011000110
00110000001110000111
00001011010001111100
00110100011010111110
11100001111010100111
11111011111011101111
01010100001101110011
00011000000001100001
00101100100000010011
11101111111100101010
11000110001110101101
10100001100111001110
11111100110010001001
00001000100011001110
11010011111000110110
01101001011111101110
00000100101010101111
00011101101110000011
11110111001011100010
01000101000110100000
10111001011110110100
