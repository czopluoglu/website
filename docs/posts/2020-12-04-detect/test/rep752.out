-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep752.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.40

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.59
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  4
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.8058
                   IDN index value:   0.7105
                           Ratio r:   0.6472

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    1    2    1    3    1    1    4    1    1
    2    4    2    4    2    4    2    4    4    2

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    2    4    6    7    9   10

   -----------CLUSTER  2-------------

    3   11   13   15   17   20

   -----------CLUSTER  3-------------

    5

   -----------CLUSTER  4-------------

    8   12   14   16   18   19
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d+++++-+----++------
   +d+++++--+----------
   ++d++++----------+--
   +++d+--+--+---++-+--
   ++++d++---+++---+---
   +++-+d-----+-++-+--+
   -++-+-d--+--+++---+-
   +--+---d+++--++-++-+
   -------+d++++-+---++
   -+----+++d+++--+--++
   ---++--+++d----+-++-
   ----++--++-d+--++-+-
   +---+-+-++-+d----+--
   +----+++-----d--+---
   ---+-++++-----d-++-+
   ---+-----+++---d+++-
   ----++-+---+-+++d+-+
   --++---+--+-+-+++d++
   ------+-++++---+-+d+
   -----+-+++----+-+++d
 
 No cross validation for this DETECT run
